### DISCUSSION TITLE
How to select a occurence  and edit it?

### DISCUSSION THREAD

**Discussion by hongnod** [dail8859/NotepadNext#581] [Q&A]
![1719497043064](https://github.com/dail8859/NotepadNext/assets/6680267/f687f217-5162-4385-bdeb-7e38c30125c1)
In NPN, I can highlight  a occurence like in the pic, I didn't find how select and edit it in parallel.

**Comment by dail8859** [comment]
`Edit > Select Next Instance (Ctrl + D)`

It could probably be improved but that is the basic functionality.

**Reply by hongnod** [reply]
Ctrl+D only one by one. it is not the same feature.

**Reply by dail8859** [reply]
Which is why I said it could be improved.

**Reply by dail8859** [reply]
Is the desire to select all the instances that were highlighted with the smart highlighter? (The green markers)
